### DISCUSSION TITLE
Initialization - Error: bot not found

### DISCUSSION THREAD

**Discussion by TommasoCere** [go-telegram-bot-api/telegram-bot-api#595] [Q&A]
This is my main function.
My problem is `var` that always return nill.
```
const TELEGRAM_APITOKEN = "MYTOKENfromBOTFATHER"

func setTelegramBot() {
	bot, err := tgbotapi.NewBotAPI(os.Getenv(TELEGRAM_APITOKEN)) //Problem
	fmt.Println(bot)           // output: <nil>
	if err != nil {
		log.Fatal(err)     // output: Not Found
	}
	bot.Debug = true
	updateConfig := tgbotapi.NewUpdate(0)
	updateConfig.Timeout = 60
	msg := tgbotapi.NewMessage(0, "Hello World")
	if _, err := bot.Send(msg); err != nil {
		log.Fatal(err)
	}
}
```

What i'm forgetting

**Comment by myandin** [comment]
func setTelegramBot() {
	bot, err := tgbotapi.NewBotAPI("MYTOKENfromBOTFATHER") //<<< NO Problem
	fmt.Println(bot)           // output: <nil>
	if err != nil {
		log.Fatal(err)     // output: Not Found
	}
	bot.Debug = true
	updateConfig := tgbotapi.NewUpdate(0)
	updateConfig.Timeout = 60
	msg := tgbotapi.NewMessage(0, "Hello World")
	if _, err := bot.Send(msg); err != nil {
		log.Fatal(err)
	}
}
